eng/pipelines/templates/Build.template.yml

```diff
@@ -15,6 +15,12 @@ steps:
 
 - template: ../tasks/MicroBuildPrefastPlugin.yml
 
+- template: ../tasks/APIScan.yml
```

Comment:
nit: APIScan and BinSkim should be in CodeAnalysis.template.yml, you can keep `"CSharp.yml"` here.